src/components/Chart/Maps.js

```diff
@@ -39,30 +52,52 @@ const symbolShapes = {
 }
 const shapes = Object.keys(symbolShapes).concat('marker')
 
-const Points = ({data, colorScale, colorAccessor, project, shape, sizes, domain}) => {
+const Points = ({data, colorScale, colorAccessor, project, shape, sizeRangeMax, hoverPoint, setHoverPoint}) => {
+
   const marker = shape === 'marker'
   let symbolPath
   if (!marker) {
-    const size = scaleOrdinal().domain(domain).range(sizes)
+    const size = scaleLinear().domain(extent(data.map(d => d.value))).range([0, sizeRangeMax])
```

Comment:
Add zero to the domain:

```suggestion
    const size = scaleLinear().domain([0, max(data.map(d => d.value))]).range([0, sizeRangeMax])
```

Makes Mumbai with 1t visible